enterpriseknowledgegraph/search/search_public_kg_sample.py

```diff
@@ -0,0 +1,84 @@
+# pylint: disable=too-many-arguments
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START enterpriseknowledgegraph_search_public_kg]
+
+from typing import Sequence
+
+from google.cloud import enterpriseknowledgegraph as ekg
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_GRAPH_LOCATION'      # Values: 'global'
+# search_query = 'YOUR_SEARCH_QUERY'
+# languages = ['en']                    # Optional: List of ISO 639-1 Codes
+# types = ['']                          # Optional: List of schema.org types to return
+# limit = 20                            # Optional: Number of entities to return
```

Comment:
If these are optional, should they be labelled `Optional` below in the sample for `search_public_kg_sample`?

Reply:
I'll set the default value as None